server/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "prisma-vscode-server",
+  "description": "Implementation of a language server in node.",
+  "version": "0.0.32",
+  "author": "Prisma",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10"
+  },
+  "prisma": {
+    "version": "76857c35ba1e1764dd5473656ecbbb2f739e1822"
+  },
+  "repository": {
+    "type": "github",
+    "url": "https://github.com/prisma/vscode"
+  },
+  "scripts": {
+    "format": "prettier --write .",
+    "lint": "eslint --fix --ext .js,.ts ."
+  },
+  "dependencies": {
+    "@prisma/fetch-engine": "^2.0.0-beta.2",
```

Comment:
I suggest pinning dependencies by removing the caret `^` to make sure it won't accidentally fetch a newer version and break your code.

In your local development workflow, you can use `yarn upgrade-interactive --latest` to update these dependencies once in a while.